lib/fencing/st_client.c

```diff
@@ -1421,6 +1431,8 @@ xml_to_event(xmlNode *msg, pcmk__action_result_t *result)
 static void
 event_free(stonith_event_t * event)
 {
+    struct event_private *event_private = event->opaque;
```

Comment:
guess the compiler doesn't make any difference if it is an assignment to an existing variable or a definition.
bit of inconsistent to once use implicit cast and then explicit cast - on the other hand if the only intention is to
have the type visible somewhere in that line it is shorter to use explicit cast in the assignment and implicit
in the definition that already has the type in the beginning.